### PR TITLE
Bug 2099795: README file for helm charts encoded in Chinese shows messy characters in the UI

### PR DESCRIPTION
**Pull request by debsmita1**
**Fixes:**
https://bugzilla.redhat.com/show_bug.cgi?id=2099795

**Root analysis:**
The chinese encoded string is not decoded to valid utf8 string

**Solution description:**
Using js-base64's decode to decode the readme contents

**Screenshot:**
Screenshot 2022-07-26 at 8 34 51 PM

**Test setup:**
1. Add https://raw.githubusercontent.com/debsmita1/my-helm-repo/main/charts/repo/community/index.yaml chart repository
2. Click on `Aqua Enforcer` helm chart to view the Readme on the sidepanel

### CODE REVIEW

File: frontend/packages/helm-plugin/src/catalog/utils/catalog-utils.tsx [hunk 1]
```diff
@@ -113,9 +113,11 @@ export const normalizeHelmCharts = (
 
         const detailsDescriptions = [
           {
+            label: t('helm-plugin~Description'),
             value: <p>{description}</p>,
           },
           {
+            label: t('helm-plugin~README'),
```
Comment (jerolimov) [inline frontend/packages/helm-plugin/src/catalog/utils/catalog-utils.tsx:120]: This change will result that the title `## README` is also shown when there is no README.

I don't find any chart without a README to confirm this, but I'm sure that was the reason to add it in the `HelmReadmeLoader`. You can see that README is now already displayed while it's loading.

Can you revert this change so that we are safe to not introduce another issue?

**Review by jerolimov — CHANGES_REQUESTED** [review]

File: frontend/packages/helm-plugin/src/catalog/components/HelmReadmeLoader.tsx [hunk 2]
```diff
@@ -26,11 +27,11 @@ const HelmReadmeLoader: React.FC<HelmReadmeLoaderProps> = ({ chartURL }) => {
       }
 
       const readmeFile = chartData?.files?.find((file) => file.name === 'README.md');
-      const readmeData = readmeFile?.data && atob(readmeFile?.data);
+      const readmeData = readmeFile?.data && Base64.decode(readmeFile?.data);
 
       if (!unmounted) {
         setLoaded(true);
-        readmeData && setReadme(t('helm-plugin~## README\n{{readmeData}}', { readmeData }));
+        readmeData && setReadme(`## ${t('helm-plugin~README\n{{readmeData}}', { readmeData })}`);
```
Comment (debsmita1) [inline frontend/packages/helm-plugin/src/catalog/components/HelmReadmeLoader.tsx:34]: did this to handle the translation

File: frontend/packages/helm-plugin/locales/ko/helm-plugin.json [hunk 3]
```diff
@@ -19,7 +19,6 @@
   "Rollback": "롤백",
   "Alternatively, developers can <2>try to configure their own custom Helm Chart repository</2>.": "또는 개발자가 <2>고유한 사용자 정의 Helm 차트 리포지토리</2>를 구성할 수 있습니다.",
   "Browse for charts that help manage complex installations and upgrades. Cluster administrators can customize the content made available in the catalog.<1></1>": "복잡한 설치 및 업그레이드를 관리하는 데 도움이 되는 차트를 살표보십시오. 클러스터 관리자는 카탈로그에 제공된 콘텐츠를 사용자 지정할 수 있습니다.<1></1>",
-  "## README\n{{readmeData}}": "## README\n{{readmeData}}",
```
Comment (debsmita1) [inline frontend/packages/helm-plugin/locales/ko/helm-plugin.json:22]: removed it because these translation files already have the `README` key

**Review by jerolimov — APPROVED** [review]
Tested it locally and works as expected. No README case works also fine again (HC: BkS400 Helm chart for Kubernetes). :+1:

/lgtm
/approve
